entries/a/finary.com.json

```
@@ -0,0 +1,13 @@
+{
+  "Finary": {
+    "domain": "finary.com",
+    "tfa": [
+      "totp"
+    ],
+    "keywords": [
+      "investing"
+    ],
+    "documentation": "https://help.finary.com/fr/article/activer-lauthentification-a-double-facteur-2fa-22gjos/",
+    "recovery": "https://help.finary.com/fr/article/jai-perdu-lacces-a-mon-2fa-google-authenticator-137rf5p/"
```

The english versions should be more useful here:
```suggestion
    "documentation": "help.finary.com/en/article/18z52w/",
    "recovery": "help.finary.com/en/article/51poms/"
```

---

@phallobst I think you accidentally cut out the protocol declaration 😜

---

Oops...